### ISSUE TITLE
Too many Sleuthkit processes with multiple evidences

### ISSUE THREAD

**Issue by tc-wleite** [sepinf-inc/IPED#397]
While working in a case with many evidence files (18 E01 images of HDDs), I ran into excessive memory consumption.
Digging into the problem, I noticed that there were 216 SleuthkitServer processes running.
I had the "robustImageReading" option enabled.

So it seems that instead of starting 12 servers (1/4 of the number of logical processors available), it is starting 12 processes for each evidence (in this case, 12 x 18 = 216).

@lfcnassif, is this the desired behavior?
Can you make a quick test (processing more than one E01 in a single command) and check if you can reproduce this behavior?
I tested with 3.18.x releases and the current master branch, and got the same result.

I think it would make more sense having a single startup of such SleuthKit processes.
I made a quick fix, if this behavior was not intentional.

The processing including the suggested fix is still running, but it seems faster, as there is more free memory to be used as disk cache.
Of course it is possible to process fewer evidences (maybe just one) at a time (with --append after the first run), so it is not critical, but sometimes is more convenient/faster to run a single processing command.

**Comment by lfcnassif** [comment]
No, it is not expected behaviour! Thanks @tc-wleite for digging into this!

Will try to reproduce and test the fix, thank you!

**Comment by lfcnassif** [comment]
Merged, did a minor change to the fix. Thank you!
